assignments/array-methods.js

```diff
@@ -54,23 +54,34 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+
+runners.forEach(person => {
+    fullName.push(person.first_name + " " + person.last_name)
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
-console.log(allCaps); 
+
+allCaps = runners.map(runner => runner.first_name.toUpperCase());
```

Comment:
Aundrea your map callback function is correct but you have to populate the `allCaps` array with the names using `.push` like you did in the first challenge so you don't return something for every instance of `.map` and just console log the `allCaps` array once. Also if you dont push it inside an array you are not storing the values your map is returning them but since their are not being stored you won't be able to access them later